### ISSUE TITLE
unable to use

### ISSUE THREAD

**Issue by Owusu-Wilson** [rdhox/react-native-smooth-picker#48]
The implementation examples given is very poor

**Comment by ouroboroscoding** [comment]
Agreed, I spent 2 hours trying to implement this, only to find out it doesn't work at all. I scroll down (to go to the top of the list) and end up with at least a hundred events on a list of five items, only to end up at the bottom of the list again. Completely unusable.
